README.md

```diff
@@ -1,24 +1,30 @@
-# README
+# govwifi-admin
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Purpose
 
-Things you may want to cover:
+This is the User Admin UI for the [GovWiFi][link_govwifi] project.
 
-* Ruby version
+## How to install and use
 
-* System dependencies
+```shell
+make build
+make serve
+```
 
-* Configuration
+Other useful Makefile targets are:
 
-* Database creation
+- `make bash` - run a shell in the docker composed application
+- `make stop` - stops the application and docker removes volumes
+- `make lint` - runs the linter
```

Comment:
Think it's worth noting here that this uses [govuk-lint](https://github.com/alphagov/govuk-lint)?